prometheus-ksonnet/lib/prometheus.libsonnet

```diff
@@ -40,7 +42,7 @@ k {
     container.withArgs([
       "-v", "-t", "-p=/etc/prometheus",
       "curl", "-X", "POST", "--fail", "-o", "-", "-sS",
-      "http://localhost:%s%s-/reload" % [$._config.prometheus_port, $._config.prometheus_path],
+      "http://localhost:%s/-/reload" % [$._config.prometheus_port],
```

Comment:
Instead of removing this, could you set it to `$._config.prometheus_web_route_prefix` (and remove the slash)?  That should have the same effect.